lib/index.ts

```diff
@@ -37,8 +37,10 @@ export interface SDPInfoParams {
 	candidates?: CandidateInfoLike[];
 	/** Capabilities for each media type */
 	capabilities?: Capabilities;
-
+	/** Crypto info objet*/
 	crypto?: CryptoInfoLike;
+	/** Generate unified like media ids*/
+	unified: boolean
```

Comment:
I think you meant for uinified to be optional

```suggestion
	/** Crypto info object */
	crypto?: CryptoInfoLike;
	/** Generate unified like media ids*/
	unified?: boolean;
```